theme/connector/view/_InfoPane.scss

```diff
@@ -2,8 +2,6 @@
   background-color: $white;
   font-size: 12px;
   padding: 24px 16px!important;
-  margin: 6px!important;
-  width:230px!important;
```

Comment:
This was for the info pane layout issues. I'm not sure why these overrides were added in the first place instead of just letting the component lay itself out within it's containers.